.github/workflows/deploy-indexer.yaml

```diff
@@ -77,6 +77,17 @@ jobs:
       RAILWAY_TOKEN: ${{ secrets.RAILWAY_TOKEN }}
     steps:
       - uses: actions/checkout@v3
+      - name: Install Node.js and pnpm
+        run: |
+          apk add --no-cache nodejs npm
+          npm install -g pnpm@10.10.0
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+      - name: Prune DB inactive schemas
+        if: ${{ needs.changes.outputs.indexer == 'true' }}
+        run: |
```

Comment:
Database pruning operations in CI/CD without proper error handling could cause deployment failures. The db:prune command should include error handling and rollback mechanisms in case of failures.